auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -242,7 +251,7 @@ export default class AffiliatedEntityTable extends Vue {
     switch (true) {
```

Comment:
What the...?

Can you fix this please?

Reply:
Can you elaborate?

Reply:
This is just another technique to use when you have varying conditions not tied to a single type of value.
https://medium.com/trabe/using-switch-true-in-javascript-986e8ad8ae4f

I'm open to suggestions but personally i don't see anything wrong with this.

Reply:
I see.

I'd prefer to see "if then else-if then else" here as it's less obtuse.

Reply:
I disagree, i feel like 2-3 layers of if - else - if - else is obtuse 🤷

Reply:
Agree to disagree 😁